### ISSUE TITLE
Previous button

### ISSUE THREAD

**Issue by airdward** [sammy6378/Multi-step-form#1]
Hey man, this is awesome, but you could add a previous button to go back and make edits to previous choices.

**Comment by sammy6378** [comment]
> Hey man, this is awesome, but you could add a previous button to go back and make edits to previous choices.

Yeah sure that's a good idea ,I probably need to add by these week, thanks alot
